facebook-commerce-events-tracker.php

```diff
@@ -958,6 +973,44 @@ private function get_cart_total() {
 			return WC()->cart ? WC()->cart->total : 0;
 		}
 
+		/**
+		 * Gets advanced matching information from a given order
+		 *
+		 * @return array
```

Comment:
We should add since tags to the new methods and classes:

```suggestion
		 * @since 2.0.2-dev.1
		 *
		 * @return array
```